lte/gateway/c/session_manager/test/test_set_session_manager_handler.cpp

```diff
@@ -549,7 +551,9 @@ TEST_F(SessionManagerHandlerTest, test_PDUStateChangeHandling) {
   session_enforcer->m5g_pdr_rules_change_and_update_upf(
       session, magma::PdrState::IDLE);
 
-  session_enforcer->m5g_send_session_request_to_upf(session);
+  RulesToProcess pending_activation, pending_deactivation;
+  session_enforcer->m5g_send_session_request_to_upf(
+      session, pending_activation, pending_deactivation);
```

Comment:
What is the purpose of adding those lines. 
There is no asserts checking what is the result of this action.

thake a look [here](https://sourcegraph.com/-/editor?remote_url=git%40github.com%3Auri200%2Fmagma.git&branch=master&file=lte%2Fgateway%2Fc%2Fsession_manager%2Ftest%2Ftest_local_enforcer.cpp&editor=JetBrains&version=v1.2.1&utm_product_name=CLion&utm_product_version=2021.1.2&start_row=0&start_col=2&end_row=0&end_col=2) to see some examples how it is done on 4g.

Reply:
The existing test cases of UT (Those are incomplete) are taking care on Task_8713. We have already done changes as part of 8713 task.

Reply:
if you use # and number of an issue it creates the link automatically. I modified your comment so it is easy accessible 

Who is going to take care of #8713 ? 
When is this going to be started?